src/mbed_tools/cli/project_management.py

```diff
@@ -97,4 +81,24 @@ def deploy(path: str, force: bool) -> None:
     REVISION: The revision of the Mbed project to check out.
     """
     click.echo("Checking out all libraries to revisions specified in .lib files. Resolving any unresolved libraries.")
-    deploy_project(pathlib.Path(path), force)
+    root_path = pathlib.Path(path)
+    deploy_project(root_path, force)
+    libs = get_known_libs(root_path)
+    _print_dependency_table(libs)
+
+
+def _print_dependency_table(libs: List) -> None:
+    click.echo("The following library dependencies were fetched: \n")
+    table = []
+    for lib in libs:
+        table.append(
+            [
+                lib.reference_file.stem,
+                lib.get_git_reference().repo_url,
+                lib.source_code_path,
+                lib.get_git_reference().ref,
+            ]
+        )
+
+    headers = ("Library Name", "Repository URL", "Location", "Git Reference")
```

Comment:
Do you think "Path" is a more clear name than "Location"? To me, "Location" could mean remote location, if it weren't for the URL also being listed here.

Reply:
Chenged name to "Path".